hack/run-functests.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
 
-GINKGO_SUITS=${GINKGO_SUITS:-"test/e2e/performanceprofile/functests"}
+GINKGO_SUITS=${GINKGO_SUITS:-"test/e2e/performanceprofile/functests/0_config test/e2e/performanceprofile/functests/1_performance test/e2e/performanceprofile/functests/6_mustgather_testing"}
 LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-"false"}
 OC_TOOL="${OC_TOOL:-oc}"
 
 which ginkgo
 if [ $? -ne 0 ]; then
-	echo "Downloading ginkgo tool"
-    go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.6.1
-    ginkgo version
+  echo "Downloading ginkgo tool"
```

Comment:
Here can we use the below instead of which , as which is deprecated against command which is builtin bash command. 

`if [[ $(command -v ginkgo) ]]; then`